### ISSUE TITLE
Add user avatar stats circle

### ISSUE THREAD

**Issue by robinschaaf** [zooniverse/mobile#36]
Add the colored circle around the user avatar like in the mockup.  Can reference PFE code for it:

![iphone 6 copy 34](https://cloud.githubusercontent.com/assets/1033477/19523007/17f43084-9611-11e6-809f-6138aab9b0dc.png)

**Comment by rogerhutchings** [comment]
Is there a similar circular ribbon used in PFE? I could only find the horizontal one... If there is, we should probably abstract it out to it's own repo rather than duplicate it.

**Comment by srallen** [comment]
A circle ribbon is coming in with the new signed in home page: https://github.com/zooniverse/Panoptes-Front-End/pull/2979/files#diff-77c5c6c59a6072f2ce08f901dccfc733R1

**Comment by robinschaaf** [comment]
So, I ended up working on this last week, and it requires an external library since react-native doesn't support SVGs yet itself.  Since the actual display of it is coded differently I don't think we could share code for that part, but most definitely the path and color calculations could share code (I ended up literally copying and pasting that code from pfe).  I don't know if it would make sense to have a shared utilities repo? @rogerhutchings ?
